py3dbp/main.py

```diff
@@ -157,6 +157,23 @@ def add_item(self, item):
 
         return self.items.append(item)
 
+    def clear_bins(self, type='a'):
+        """Clear Packer bins. Argument type: 'a' = all (default) | 'f' = fitted | 'u' = unfitted """
```

Comment:
`all`, `fitted` and `unfitted` would be perfect. Think about code as self-documenting.
Compare invocations : 
`packer.clear_bins(type='fitted')`

vs

`packer.clear_bins(type='f')`

Avoid using such short type and variable names - it is later hard to read and understand code.

Reply:
Thanks! Changed.